APIConsumer/APIConsumer.html

```diff
@@ -0,0 +1,32 @@
+﻿<!DOCTYPE html>
```

Comment:
I would advise to rename folder name with `http-clients` or `api-consumer`, it's better to have folders with _lowercase_ or _kebab-case_ for .Net projects that are not referred directly to class libraries